bindings/pydeck/pydeck/io/html.py

```diff
@@ -87,14 +88,30 @@ def display_html(filename):
 
 
 def iframe_with_srcdoc(html_str, width="100%", height=500):
-    width = '"{}"'.format(width) if type(width) == str else width
-    iframe = """<iframe src="about:blank" frameborder="0" srcdoc="{}" width={} height={}></iframe>""".format(
-        html.escape(html_str), width, height
+    if isinstance(width, str):
+        width = '"{}"'.format(width)
+
+    iframe = """
+        <iframe
+            width={width}
+            height={height}
+            frameborder="0"
+            srcdoc="{srcdoc}"
+        ></iframe>
+    """
```

Comment:
Likewise
```suggestion
    iframe = f"""
        <iframe
            width={width}
            height={height}
            frameborder="0"
            srcdoc="{html.escape(html_str)}"
        ></iframe>
    """
```

Reply:
I couldn't do this because pasting in the html makes it so that the `dedent` no longer works as intended. But that's all just to keep the string formatting pretty. If we don't care about the string formatting, we can just use f-strings. :)

Reply:
I agree with Kyle here, let's prefer f-strings.

Reply:
done!